flask_iiif/api.py

```diff
@@ -206,6 +201,8 @@ def resize(self, dimensions, resample=None):
             )
 
         arguments = dict(size=(width, height), resample=resample)
+        if resample:
+            arguments["resample"] = resample
```

Comment:
This is not needed given the line above, right? `arguments` will already have a `"resample"` key set to the value passed as argument to the `resize` function now that the `resample` assignment from the setting has been removed.

Reply:
Oops, meant to take it out from the line above - I don't wnt to pass "resample=None" to resize.

Reply:
or better yet:

```python
arguments = {...}  # without resample
if resample is not None:
    arguments["resample"] = resample
```